app.js

```diff
@@ -25,19 +33,68 @@ function checkResponseNo(response) {
 userResponse = prompt('Does Kyle like rain drops on roses?');
 console.log('Does Kyle like rain drops on roses?');
 checkResponseYes(userResponse);
+numQuestions++;
 
 userResponse = prompt('Does Kyle like whiskers on kittens?');
 console.log('Does Kyle like whiskers on kittens?');
 checkResponseYes(userResponse);
+numQuestions++;
 
 userResponse = prompt('Does Kyle like beer?');
 console.log('Does Kyle like beer?');
 checkResponseYes(userResponse);
+numQuestions++;
 
 userResponse = prompt('Does Kyle like sweet potatoes?');
 console.log('Does Kyle like sweet potatoes?');
 checkResponseNo(userResponse);
+numQuestions++;
 
 userResponse = prompt('Does Kyle like sleep?');
 console.log('Does Kyle like sleep?');
 checkResponseYes(userResponse);
+numQuestions++;
+
+for(var i = 0; i < 4 && notGuessed; i++){
+  userGuess = prompt('Guess Kyle\'s favorite number.');
+  console.log('Guess Kyle\'s favorite number.');
+
+  if(userGuess > favoriteNum){
+      alert('Too high!');
+      console.log('Too High!');
+  } else if (userGuess < favoriteNum) {
+    alert('Too low!');
+    console.log('Too low!');
+  } else {
+    alert('You got it!');
+    console.log('You got it!');
+    notGuessed = false;
+    numCorrect++;
+  }
+}
+numQuestions++;
+
+notGuessed = true;
+for(var i = 0; i < 6 && notGuessed; i++){
+  userGuess = prompt('Guess a Country that Kyle has visited (other than the US).');
+  console.log('Guess a country that kyle has visited');
+
+  for(var itr = 0; itr < countriesVisited.length; itr++){
```

Comment:
you can just use j instead of itr, here, it'll do the same thing and be a bit easier to read